mongodb_migrations/config.py

```diff
@@ -22,8 +23,21 @@ class Configuration(object):
     execution = Execution.MIGRATE
     to_datetime = None
 
-    def __init__(self):
-        self._from_ini()
+    def __init__(self, config: Dict[str, Any]=None):
```

Comment:
By using type here, we need to update setup.py to explicitly declare that this package is python3 only.

Reply:
Fixed, omitting typing.